cli/src/mdastToRst.ts

```diff
@@ -201,6 +201,12 @@ const visitors: {
     c.add(children, (text) => text.replace(/\*/g, "\\*"));
     c.add("*");
   },
+  example(c, n) {
```

Comment:
For some reason I can't see where you're actually using this?

Reply:
Not using it at all now -- I was originally using it for each individual method detail, but now i'm using tables instead.